woocommerce/woocommerce-ios
# [Shipping labels] Delete package (networking)

Closes: #14589


## Description


- Added deletePackage action, for now it only supports the custom packages
- When backend adds support for deleting/unstaring predefined packages (M2 or M3) we will add support for that with https://github.com/woocommerce/woocommerce-ios/issues/14679

## Testing information


- Install and set up the Woo Shipping extension on your store.
- Build and run the app with the revampedShippingLabelCreation feature flag enabled.
- Create an order with the processing status and at least one physical product.
- In your WooCommerce product settings on the web, change the weight and dimension units and save your changes.
 On the Orders tab in the app, open the order you created.
- In the order details, select "Create Shipping Label."
- Tap "Select a Package."
- Switch to Saved tab
- If you do not have any saved packages go to Custom tab and add one
- Swipe on one of the Saved packages and tap trash icon to delete it
- Check that the package is removed from the list
- Pull to refresh the list and check that the removed package is still not in the list

## Screenshots


https://github.com/user-attachments/assets/6f5cf4fd-cfc4-4f1a-8cb8-d7a19e6217fe

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
